scripts/utils/dragging.ts

```diff
@@ -0,0 +1,83 @@
+import {IArticle} from 'superdesk-api';
+
+/**
+ * NOTICE:
+ * if you want to test drag image in chrome, make sure to close dev tools first,
+ * otherwise this thumbnail image might not be displayed, depends on some performance settings there.
```

Comment:
What do you mean by performance settings? Also, if you initially have dev tools open before rendering the thumbnail, when you render it, wouldn't it get resized accordingly?

Reply:
I've moved this file so comment was written probably a few years ago. I don't remember what are the performance settings.

> Also, if you initially have dev tools open before rendering the thumbnail, when you render it, wouldn't it get resized accordingly?

I don't get what you mean about resizing. That thumbnail is rendered in full size and is hidden, so there's never a reason to resize it. When you drag another element, you can set it as drag image. see https://codepen.io/webgeeker/full/KBzrxE/